src/main/python/keycodes.py

```diff
@@ -556,6 +539,11 @@ def generate_keycodes_for_mask(label, mask):
         lbl = "M{}".format(x)
         KEYCODES_MACRO.append(Keycode(0x5F12 + x, lbl, lbl))
 
+    KEYCODES_USER.clear()
+    for x in range(len(keyboard.custom_keycodes)):
```

Comment:
`for x, key in enumerate(keyboard.custom_keycodes)`

Reply:
Didn't know about enumerate, thats really usefull, I've added it to the code